Linux Script/configure-linux.sh

```diff
@@ -81,743 +81,739 @@ LOGGLY_DISTRIBUTION_ID="41058"
-	createRsyslogDir
-
-	if [ "$LINUX_DO_VERIFICATION" = "true" ]; then
-		#check if the logs are going to loggly fro linux system now
-		checkIfLogsMadeToLoggly
-	fi
-
-	if [ "$IS_INVOKED" = "" ]; then
-		logMsgToConfigSysLog "SUCCESS" "SUCCESS: Linux system successfully configured to send logs via Loggly."
-	fi
-
+  #log message indicating starting of Loggly configuration
+  logMsgToConfigSysLog "INFO" "INFO: Initiating Configure Loggly for Linux."
+    
+  if [ "$LINUX_ENV_VALIDATED" = "false" ]; then
+  checkLinuxLogglyCompatibility
+  fi
+  
+  #if all the above check passes, write the 22-loggly.conf file
+    checkAuthTokenAndWriteContents
+  
+  #create rsyslog dir if it doesn't exist, Modify the permission on rsyslog directory if exist on Ubuntu
+    createRsyslogDir
+  
+  if [ "$LINUX_DO_VERIFICATION" = "true" ]; then
+  #check if the logs are going to loggly fro linux system now
+    checkIfLogsMadeToLoggly
+  fi
+  
+  if [ "$IS_INVOKED" = "" ]; then
+  logMsgToConfigSysLog "SUCCESS" "SUCCESS: Linux system successfully configured to send logs via Loggly."
+  fi
+  
 }
 
 #remove loggly configuration from Linux system
 removeLogglyConf()
 {
-	#log message indicating starting of Loggly configuration
-	logMsgToConfigSysLog "INFO" "INFO: Initiating uninstall Loggly for Linux."
-
-	#check if the user has root permission to run this script
-	checkIfUserHasRootPrivileges
-
-	#check if the OS is supported by the script. If no, then exit
-	checkIfSupportedOS
-
-	#set the basic variables needed by this script
-	setLinuxVariables
-
-	#remove systemd-rsyslog configuration
-	revertSystemdChanges
-
-	#remove 22-loggly.conf file
-	remove22LogglyConfFile
-
-	#restart rsyslog service
-	restartRsyslog
-
-	#log success message
-	logMsgToConfigSysLog "SUCCESS" "SUCCESS: Uninstalled Loggly configuration from Linux system."
+  #log message indicating starting of Loggly configuration
+  logMsgToConfigSysLog "INFO" "INFO: Initiating uninstall Loggly for Linux."
+    
+  #check if the user has root permission to run this script
+    checkIfUserHasRootPrivileges
+  
+  #check if the OS is supported by the script. If no, then exit
+    checkIfSupportedOS
+  
+  #set the basic variables needed by this script
+  setLinuxVariables
+  
+  #remove systemd-rsyslog configuration
+  revertSystemdChanges
+  
+  #remove 22-loggly.conf file
+  remove22LogglyConfFile
+  
+  #restart rsyslog service
+  restartRsyslog
+  
+  #log success message
+  logMsgToConfigSysLog "SUCCESS" "SUCCESS: Uninstalled Loggly configuration from Linux system."
 }
 
 #checks if user has root privileges
-checkIfUserHasRootPrivileges()
+  checkIfUserHasRootPrivileges()
 {
-	#This script needs to be run as root
-	if [[ $EUID -ne 0 ]]; then
-	   logMsgToConfigSysLog "ERROR" "ERROR: This script must be run as root."
-	   exit 1
-	fi
+  #This script needs to be run as root
+  if [[ $EUID -ne 0 ]]; then
```

Comment:
What was wrong with the formatting of the indents before? I noticed that you are changing it but is there any particular reason?